### ISSUE TITLE
Time Clock-in Clock-out Feature Request.

### ISSUE THREAD

**Issue by JarbyDev** [sapplica/sentrifugo#48]
Will be nice Hourly Employees have the ability to Time clock-in and clock-out in a daily Basis.

**Comment by ausiddiqui** [comment]
or some API support to feed in time-in, time-out from another system.

**Comment by mahavidhan** [comment]
If there is a biometric device data available and if the vendor is willing to push the clock-in/out data then it would be great if that can be pulled from sentrifugo to maintain an updated attendance tracker.

**Comment by sapplica** [comment]
Hi,

Currently, it's not available in Sentrifugo. If you would like to have any customization done, kindly contact our sales team at sales@sentrifugo.com Thank you!

**Comment by csewardsr** [comment]
Good idea.  Not a hard feed to import into a punch table, then wait for a matching punch and send calculated hours to the timecard.
